src/App.vue

```diff
@@ -86,8 +86,8 @@
                 {{ group.name }}
               </h2>
               <Service
-                v-for="(item, index) in group.items"
-                :key="`service-${groupIndex}-${index}`"
+                v-for="(item) in group.items"
+                :key="`${currentPage}-${group.name}-${item.name}`"
```

Comment:
Goup and item name can contain anything like emoji and all kind of char, it might be safer to keep the index, just adding the currentPage prefix ?